pymodbus/server/async_io.py

```diff
@@ -49,6 +49,8 @@ def __init__(self, owner):
         self.receive_queue: asyncio.Queue = asyncio.Queue()
         self.handler_task = None  # coroutine to be run on asyncio loop
         self.framer: ModbusFramer
+        self.request_tasks: set[asyncio.Task] = set()
```

Comment:
This is opening for parallel processing in the server, that is one big can of worms, please do not open that.

Reply:
I'll remove use of set